cli/command/plugin/install.go

```diff
@@ -18,16 +18,20 @@ type pluginOptions struct {
 	name       string
 	grantPerms bool
 	disable    bool
+	args       []string
 }
 
 func newInstallCommand(dockerCli *command.DockerCli) *cobra.Command {
 	var options pluginOptions
 	cmd := &cobra.Command{
-		Use:   "install [OPTIONS] PLUGIN",
+		Use:   "install [OPTIONS] PLUGIN [key=value...]",
```

Comment:
In that case the "key" is `name.field`, so not sure we should complicate it further. We should probably explain it in other parts of the docs